storage/src/vespa/storage/distributor/persistencemessagetracker.cpp

```diff
@@ -28,41 +29,42 @@ PersistenceMessageTrackerImpl::PersistenceMessageTrackerImpl(
       _revertTimestamp(revertTimestamp),
       _trace(_reply->getTrace().getLevel()),
       _requestTimer(node_ctx.clock()),
-      _cancel_scope(),
+      _cancel_scope(cancel_scope),
       _n_persistence_replies_total(0),
       _n_successful_persistence_replies(0),
       _priority(_reply->getPriority()),
       _success(true)
 {
 }
 
-PersistenceMessageTrackerImpl::~PersistenceMessageTrackerImpl() = default;
+PersistenceMessageTracker::~PersistenceMessageTracker() = default;
 
-void
-PersistenceMessageTrackerImpl::cancel(const CancelScope& cancel_scope)
-{
-    _cancel_scope.merge(cancel_scope);
-}
-
-void
-PersistenceMessageTrackerImpl::prune_cancelled_nodes_if_present(
+bool
```

Comment:
consider enum here to better communicate the meaning of true and false

Reply:
Done